docs/en/flash.md

```diff
@@ -28,7 +28,9 @@ Run the following command to flash an *aggregated* binary as is produced for exa
 - `size` is given in bits. Specify `4m` for 512&nbsp;kByte and `32m` for 4&nbsp;MByte.  If unsure, try a smaller size;
```

Comment:
Oh nice, just found the [PR](https://github.com/espressif/esptool/pull/131), thanks. Don't you think it'd make sense to mention that only for [esptool.py >= 1.2](https://github.com/espressif/esptool/releases) you can skip the flash size parameter?